plugins/dialog/src/commands.rs

```diff
@@ -121,12 +121,13 @@ pub(crate) async fn open<R: Runtime>(
         #[cfg(desktop)]
         {
             if options.multiple {
-                let folders = dialog_builder.blocking_pick_folders();
-                if let Some(folders) = &folders {
+                let mut folders = dialog_builder.blocking_pick_folders();
```

Comment:
I think I'd rather avoid mutating here and instead use `.iter().map` later at line 133.

Reply:
Like this? I don't think this works that well for the other 2 below that return FileResponse structs but take a look.

https://github.com/tauri-apps/plugins-workspace/pull/1168/commits/68e7d1ebf5a1f02c67cd4dcd89c65223341402d2